src/Blorc.OpenIdConnect.Tests/NavigateEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Blorc.OpenIdConnect.Tests
+{
+    using Microsoft.AspNetCore.Components;
+
+    public class NavigateEventArgs
```

Comment:
the option is called NavigationOptions. Shall we rename event args to NavigationEventArgs?